File: java/src/main/java/com/meteroid/models/TaxExemptionType.java
// This file is @generated
package com.meteroid.models;

import com.fasterxml.jackson.annotation.JsonValue;
import com.meteroid.Utils.ToQueryParam;

public enum TaxExemptionType implements ToQueryParam {
    REVERSE_CHARGE("REVERSE_CHARGE"),
    TAX_EXEMPT("TAX_EXEMPT"),
    NOT_REGISTERED("NOT_REGISTERED");
    private final String value;

    TaxExemptionType(String value) {
        this.value = value;
    }

    @JsonValue
    public String getValue() {
        return this.value;
    }

    @Override
    public String toQueryParam() {
        return this.value;
    }
}

File: python/meteroid/models/tax_exemption_type.py
# this file is @generated
import enum


class TaxExemptionType(str, enum.Enum):
    REVERSE_CHARGE = "REVERSE_CHARGE"
    TAX_EXEMPT = "TAX_EXEMPT"
    NOT_REGISTERED = "NOT_REGISTERED"
    EXPORT = "EXPORT"

    def __str__(self) -> str:
        return str(self.value)
